README.md

```diff
@@ -237,6 +238,7 @@ If the problems from [LeetCode](https://leetcode.com/) are not enough and you ne
     - [Hackr.io - Data Structures and Algorithms Tutorials and Courses](https://hackr.io/tutorials/learn-data-structures-algorithms) - Big collection of tutorials and courses.
     - [Classpert - Data Structures and Algorithms Courses](https://classpert.com/data-structures-and-algorithms) - List of over 80 Online Courses (free and paid).
     - [Scaler - Data Structures Tutorial](https://www.scaler.com/topics/data-structures/) - Learn in-depth about the need & applications of data structures.
+    - [InterviewBit - Data Structures and Algorithms](https://www.interviewbit.com/courses/programming/) - Lear DSA Concepts with practice problems asked at top tech companies.
```

Comment:
Please remove this, most of the "Training Sites" contains sections like this and they aren't mentioned there.

Reply:
Hey Apologies, for the massive delay in getting back, would this link be more relevant: https://www.interviewbit.com/coding-interview-questions/ or should I move the url to the section you've suggested?